components/instructions/programs/associatedTokenAccount.tsx

```diff
@@ -0,0 +1,50 @@
+import { Connection, PublicKey } from '@solana/web3.js'
+import { AccountMetaData } from '@solana/spl-governance'
+import { SPL_TOKENS } from '@utils/splTokens'
+
+const ATA_PROGRAM_ID = new PublicKey(
+  'ATokenGPvbdGVxr1b2hvZbsiqW5xWH25efTNsLJA8knL'
```

Comment:
Can this be retrieved from a 3rd party package instead?

Reply:
yes!